packages/snjs/lib/services/sync/sync_service.ts

```diff
@@ -586,6 +591,9 @@ export class SNSyncService extends PureService<
         throw Error(`Unhandled timing strategy ${useStrategy}`);
       }
     }
+    if (this.dealloced) {
+      return;
+    }
```

Comment:
Yeah, not something I wanted in either tbh. But basically this can happen if on sync completion the sync function wants to sync again, and so begins syncing, but then in a test somewhere, the application is destroyed.